test/mlir/conversion/onnx_to_torch/squeezeop/squeezeop11_unsorted_axis.onnx.mlir

```diff
@@ -0,0 +1,13 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {}  {
+  func @main_graph(%arg0: tensor<4x1x4x3x1x4x1xf32>) -> tensor<4x4x3x4xf32> attributes {input_names = ["input"], output_names = ["output"]} {
+//CHECK: %[[CONST1:.*]] = torch.constant.int 1
+//CHECK: [[RES1:%.]] = torch.aten.squeeze.dim %arg0, %[[CONST1]] : !torch.vtensor<[4,1,4,3,1,4,1],f32>, !torch.int -> !torch.vtensor<[4,4,3,4],f32>
```

Comment:
This is functionally not correct. `aten.squeeze` is only applied to the dimension you are specifying.

The result type needs to change here. Maybe look at flatten as a reference.